apiary-metastore-listener/src/test/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListenerTest.java

```diff
@@ -83,16 +86,15 @@ public void setup() {
     snsListener = new ApiarySnsListener(configuration, snsClient);
     when(snsClient.publish(any(PublishRequest.class))).thenReturn(publishResult);
 
-    FieldSchema partitionColumn1 = new FieldSchema("Column_1", "String", "");
-    FieldSchema partitionColumn2 = new FieldSchema("Column_2", "String", "");
-    FieldSchema partitionColumn3 = new FieldSchema("Column_3", "String", "");
+    FieldSchema partitionColumn1 = new FieldSchema("column_1", "String", "");
```

Comment:
OK, so I see where the values in the documentation were coming from. I think that when Hive creates these FieldSchema objects then the types are all lower case. The FieldSchema class doesn't enforce or change anything so if you create them yourself you can do what you like but I think it's best to try match what Hive does.